app/src/controllers/project/selectors.js

```diff
@@ -48,6 +48,22 @@ export const defectTypesSelector = createSelector(subTypesSelector, (subTypes) =
   ),
 );
 
+export const legendDefectTypesSelector = createSelector(subTypesSelector, (subTypes) => {
+  const PREFIX = 'statistics$defects$';
+  const result = [
+    'statistics$executions$total',
```

Comment:
Execution statistics doesn't belong to defect types. Why is it here?

Reply:
Maybe, misleading name for the selector was chosen. Something like `orderedContentFieldsSelector` will be more appropriate in this case. In legend of some widgets execution statistics is used as well as defect types statistics.